factory/glideFactoryEntry.py

```diff
@@ -1288,12 +1287,12 @@ def unit_work_v3(
             if "frontend_scitoken" in submit_credentials.identity_credentials:
                 del submit_credentials.identity_credentials["frontend_scitoken"]
         else:
-            (fd, tmpnm) = tempfile.mkstemp(dir=submit_credentials.cred_dir)
             try:
                 entry.log.info("frontend_scitoken supplied, writing to %s" % scitoken_file)
+                (fd, tmpnm) = tempfile.mkstemp(dir=submit_credentials.cred_dir)
+                with os.fdopen(fd, "w", encoding="utf-8") as f:
+                    f.write(f"{scitoken_data}\n")
```

Comment:
Do we still need to add the newline now that we don't strip the data from the classad?